app/build.gradle

```diff
@@ -9,13 +8,17 @@ android {
         applicationId "com.wikaba.ogapp"
         minSdkVersion 9
         targetSdkVersion 21
-        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
     }
 
     sourceSets {
         main {
             java.srcDirs = ['src/main/java-gen', 'src/main/java']
         }
+
+        //make AS and Gradle aware of the active dependencies
+        androidTest {
+            java.srcDirs = ['src/main/java-gen', 'src/test/java', 'src/test/resources']
+        }
```

Comment:
+1 typo here

the correct should be here:

        androidTest {
            java.srcDirs = ['src/main/java-gen']
        }

due to an issue which existed on pre-1.2.3 gradle support (it was presumably fixed in AS but I am currently running the new RC and did not inspected it)

Also note that the current tests in the Android Instrumentation Folder are Unit Test